config.default.yaml

```diff
@@ -66,10 +66,11 @@ base_network:
   min_voltage_rebase_voltage: 35000  # [V] minimum voltage in base network
 
 load_options:
-  weather_year: 2014  # Not active, but load scenarios available with different weather year (different renewable potentials)
-  prediction_year: 2030  # Not active, but load scenarios available with different prediction year (GDP, population)
+  ssp: "ssp2-2.6" # shared socio-economic pathway (GDP and population growth) scenario to consider
+  weather_year: 2013  # Load scenarios available with different weather year (different renewable potentials)
+  prediction_year: 2030  # Load scenarios available with different prediction year (GDP, population)
   scale: 1  # scales all load time-series, i.e. 2 = doubles load
-
+  region_load: "Africa" # Load demand scenarios based on a specified reference region
```

Comment:
In general, as discussed, I don't like this fixed "region_load"; this should be automatized depending on the specific country list.
Basically, we need (another) configuration file where we state for every name file (e.g. Africa, Asia etc) what are the countries that are contained and the workflow load the file according to that.
I think this PR is fine as is once the workflow works; however, most of these loading features may be incorporated into a specific script (and rule) named build_load, for example, whose goal is to merge different files and data to produce an nc file containing the demand for every bus of the workflow.
This nc file would be now an input for add_electricity.

That proposed change may help the workflow also giving users a clear understanding about what is the input file they should change if they want to bypass our procedure, for example.

Any thoughts?

Reply:
Agree. To enable linkers the add_electrcity script should just upload a .nc or csv file.

The other options for the load will end up in a `build_load` script. The "region_load" idea is excellent! I would probably orient at the ISO code instead of using something else. Soon we should have the GEGIS power and build it also according to ISO naming.

Regarding the current code, I will just debug it and merge the PR. We need to write more clearly that the IDE is essential (i.e. VS code) and should link to the hackathon to get introduced to debugging (most important).